tox.ini

```diff
@@ -39,4 +39,5 @@ deps =
     flake8-html
     flake8-debugger
 commands =
-    flake8 --format=html --htmldir={toxinidir}/flake8 --doctests src tests setup.py {posargs}
+    - flake8 --format=html --htmldir={toxinidir}/flake8 --doctests src setup.py
```

Comment:
I had to look this one up in the [tox documentation](https://tox.readthedocs.io/en/latest/config.html#conf-commands).